segmentation/config.py

```diff
@@ -114,28 +127,29 @@ def parse_main_file(self, file):
         # Replace relative path if needed
         dir_file = os.path.dirname(file)
         for key, val in struct.items():
-            if os.path.dirname(val) == '':
-                struct[key] = os.path.realpath(os.path.join(dir_file, val))
+            if isinstance(val, str):
+                if os.path.dirname(val) == '':
+                    struct[key] = os.path.realpath(os.path.join(dir_file, val))
```

Comment:
Why do you need this check?

Reply:
c'est le seul changement nécessaire, pour pouvoir appeller 
segmentation_pipeline avec -f config_all.json
(config all, c'est ecrit apres un premier passage et ca contient tous les dict) !

pas sur  encore que ce soit vraiment necessaire, mais c'était simple à faire !